mingw-w64-headers-git/PKGBUILD

```
@@ -25,7 +25,7 @@ sha256sums=('SKIP'
 
 pkgver() {
   cd "${srcdir}/mingw-w64"
-  printf "9.0.0.%s.%s" "$(git rev-list --count $_commit)" "$(git rev-parse --short $_commit)"
+  printf "10.0.0.%s.%s" "$(git rev-list --count $_commit)" "$(git rev-parse --short $_commit)"
```

```suggestion
  printf "10.0.0.r%s.%s" "$(git rev-list --count $_commit)" "$(git rev-parse --short $_commit)"
```

Could we maybe add a "r" while at it?